.kokoro/presubmit/integration.cfg

```diff
@@ -27,11 +27,6 @@ env_vars: {
   value: "secret_manager/java-it-service-account"
 }
 
-env_vars: {
```

Comment:
was this intentional?

Reply:
owlbot did this, i didn't touch the this file at all. it's in one of the "updates from owlbot" commits

Reply:
i think it's because of a mistake i made with the owlbot.py file at one point, i can fix it